var/spack/repos/builtin/packages/py-flit-core/package.py

```diff
@@ -32,11 +33,6 @@ class PyFlitCore(PythonPackage):
     version("3.0.0", sha256="a465052057e2d6d957e6850e9915245adedfc4fd0dd5737d0791bf3132417c2d")
     version("2.3.0", sha256="a50bcd8bf5785e3a7d95434244f30ba693e794c5204ac1ee908fc07c4acdbf80")
 
-    # pyproject.toml
-    depends_on("python@3.6:", when="@3.4:", type=("build", "run"))
-    depends_on("python@3.4:", when="@3:", type=("build", "run"))
-    depends_on("python@2.7,3.4:", type=("build", "run"))
```

Comment:
I'm guessing the constraints are no longer needed because the minimum version of Python available in Spack is now high enough that it doesn't matter?

Does it still need a dependency on Python as a whole? Or is that implicit for a PythonPackage subclass?

Reply:
Yes and yes.